src/main/java/jdbchomework/console/ContentsUserMenu.java

```diff
@@ -211,7 +214,12 @@ public void companyMenu() {
                             .getValidInputFromUser(PLEASE_ENTER_COMPANY_ID, InputType.INTEGER));
                     System.out.println(companyController.getCompanyById(id));
                     break;
+
                 case 4:
+                    id = Integer.parseInt(visualUserMenu.getValidInputFromUser(PLEASE_ENTER_COMPANY_ID, InputType.INTEGER));
+                    companyController.getAllCompanyDevelopers(id).forEach(System.out::println);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)